experimental/packages/opentelemetry-sdk-web/src/SessionIdSpanProcessor.ts

```diff
@@ -0,0 +1,67 @@
+
+/*
+ * Copyright The OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Context } from "@opentelemetry/api";
+import { ReadableSpan, Span, SpanProcessor } from "@opentelemetry/sdk-trace-base";
+
+const SESSION_ID_BYTES = 16;
+const SHARED_BUFFER = Buffer.allocUnsafe(SESSION_ID_BYTES);
```

Comment:
`Buffer` is a nodejs-only API, requiring polyfill in all browsers (combined with next comment, looks like you copy-pasted node version of random id generator)

also see https://sindresorhus.com/blog/goodbye-nodejs-buffer

Reply:
Example of creating browser safe IDs here: https://github.com/open-telemetry/opentelemetry-js/blob/main/packages/opentelemetry-sdk-trace-base/src/platform/browser/RandomIdGenerator.ts#L27

Reply:
I have removed the session ID processors for now, will handle it in a separate PR.